src/main/resources/application.properties

```diff
@@ -5,4 +5,7 @@ spring.datasource.username=postgres
 spring.datasource.password=11280534
 spring.jpa.hibernate.ddl-auto=create
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
-spring.jpa.generate-ddl=true
\ No newline at end of file
+spring.jpa.generate-ddl=true
+
+bot.name=AggrNewsBot
+bot.token=6607685280:AAE4RpkYysaiIZiPHyknl6RoN-q4AS0v00w
```

Comment:
Never push secret tokens to repository. Do you want it to be reused by a random internet user?